openapi-filter.js

```diff
@@ -87,10 +87,10 @@ else {
 if (argv.outfile) {
     fs.writeFileSync(argv.outfile,s,'utf8');
 
-    if(argv.verbose) console.info('OUT File:     ' + argv.outfile)
+    if (argv.verbose) console.info('OUT File:     ' + argv.outfile)
 }
 else {
     console.log(s);
 }
-if(argv.verbose) console.info('\x1b[32m%s\x1b[0m','\n','✅ OpenAPI document was filtered successful!')
+if (argv.verbose) console.info('\x1b[32m%s\x1b[0m','\n','✅ OpenAPI document was filtered successful!')
```

Comment:
For clarity of what these escape sequences mean, can we specify them as constants, and provide a mechanism for turning them off where they may not be supported? Something like https://github.com/Mermade/oas-kit/blob/95f2c890bc84d4ec251116d5421161d8b735e8ca/packages/oas-kit-common/index.js#L5

Probably 'successfully' rather than 'successful`.

Reply:
The escape sequence provides a green colouring of the output, so that it stands out.
I have removed the sequence and changed "OpenAPI" to just "Document".

This is what the output looks like for ` node openapi-filter.js ./test/configFile/input.yaml ./test/configFile/output.yaml --configFile ./test/configFile/options.json --verbose`

<img width="378" alt="CleanShot 2021-01-10 at 17 09 59@2x" src="https://user-images.githubusercontent.com/952446/104128246-bd24b080-5366-11eb-90a1-55c83ed163ab.png">

Reply:
May be an additional idea to add to the verbose output. Would you be in favour of showing a result table that shows the results of kept OpenAPI operations and the filtered out. Which can be useful for CI pipelines or for users to see what is actually impacted by the filter process. 

Like for example the Newman output:

https://cdn2.hubspot.net/hubfs/208250/Blog_Images/newman3.png

Reply:
Maybe if we could use `console.table` as a first step (and have it optional, as `--verbose` is a count, you can check whether it is `> 1` etc).